dist/api_data.py

```diff
@@ -491,7 +503,8 @@ def __ge__(self, other):
         type='string', undoc=True),
 ]
 
-layered_meta = format_meta + layered_config + log_runtime_config + connection_disaggregated_config
+layered_meta = format_meta + layered_config + log_runtime_config +\
+                connection_disaggregated_config + connection_page_delta_config
```

Comment:
The config has been moved to connection level, why we still keep it for layered layer? I think it can be removed from here.